### PR TITLE
Make governoship colors different from liege one

### PR DESCRIPTION
**Pull request by IhateTrains**
part of #90

### CODE REVIEW

File: ImperatorToCK3.UnitTests/CK3/Titles/LandedTitlesTests.cs [hunk 1]
```diff
@@ -309,5 +311,49 @@ public class LandedTitlesTests {
 
 			Assert.Equal(20, titles["c_county1"].GetDevelopmentLevel(date)); // (10+40)/2 - sqrt(25)
 		}
+
+		[Fact]
+		public void DerivedColorsHaveSameHueAsBase() {
```
Comment (Idhrendur) [inline ImperatorToCK3.UnitTests/CK3/Titles/LandedTitlesTests.cs:316]: You can probably have a more general name.

File: ImperatorToCK3.UnitTests/CK3/Titles/LandedTitlesTests.cs [hunk 2]
```diff
@@ -309,5 +311,49 @@ public class LandedTitlesTests {
...
+
+			var derivedTitle1 = titles.Add("k_derived1");
+			var derivedColor1 = titles.GetDerivedColor(baseColor);
+			derivedTitle1.Color1 = derivedColor1;
+
+			var derivedTitle2 = titles.Add("k_derived2");
+			var derivedColor2 = titles.GetDerivedColor(baseColor);
+			derivedTitle2.Color1 = derivedColor2;
+
+			Assert.Equal(baseColor.H, derivedColor1.H);
```
Comment (Idhrendur) [inline ImperatorToCK3.UnitTests/CK3/Titles/LandedTitlesTests.cs:331]: Is there a reason you're not also checking the saturations are the same?

**Review by Idhrendur — APPROVED** [review]

File: ImperatorToCK3.UnitTests/CK3/Titles/LandedTitlesTests.cs (hunk 1 above)
Comment (IhateTrains) [reply ImperatorToCK3.UnitTests/CK3/Titles/LandedTitlesTests.cs:316]: Changing to `DerivedColorsHaveCorrectComponents`.

File: ImperatorToCK3.UnitTests/CK3/Titles/LandedTitlesTests.cs (hunk 2 above)
Comment (IhateTrains) [reply ImperatorToCK3.UnitTests/CK3/Titles/LandedTitlesTests.cs:331]: No, I can add an assertion.

**Pull request merged**